### PR TITLE
Remove dependencies on external template files

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: src/alr/alr-commands-init.adb [hunk 1]
```diff
@@ -18,38 +19,144 @@ package body Alr.Commands.Init is
...
+         Put_Line ("      for Switches (""ada"") use");
+         Put_Line ("        (""-gnatVa"", ""-gnatwa"", ""-g"", ""-O2"",");
+         Put_Line ("         ""-gnata"", ""-gnato"", ""-fstack-check"");");
+         Put_Line ("   end Compiler;");
+         Put_New_Line;
+         Put_Line ("   package Binder is");
+         Put_Line ("      for Switches (""ada"") use (""-Es"");");
+         Put_Line ("   end Binder;");
+         Put_New_Line;
+         Put_Line ("end project " & Mixed_Name & ";");
```
Comment (mosteo) [inline src/alr/alr-commands-init.adb]: "end project " --> "end "

File: src/alr/alr-commands-init.adb [hunk 2]
```diff
@@ -43,12 +43,15 @@ package body Alr.Commands.Init is
       end if;
 
       declare
-         Guard : Folder_Guard (
```
Comment (mosteo) [inline src/alr/alr-commands-init.adb]: Removing this one causes a spurious 'alire' folder to be created in the CWD. Possible fix is to change line 163 from 
`OS_Lib.Create_Folder (Paths.Alr_Working_Folder);` 
to 
`OS_Lib.Create_Folder (Name / Paths.Alr_Working_Folder);`
(without undoing the deletion)

File: src/alr/alr-commands-init.adb (hunk 1 above)
Comment (pmderodat) [reply src/alr/alr-commands-init.adb]: Oops, I tested the init command but forgot to do a build after it… will fix, thanks!

File: src/alr/alr-commands-init.adb (hunk 2 above)
Comment (pmderodat) [reply src/alr/alr-commands-init.adb]: Thank you for catching this! I’ll have a look.

**Comment by mosteo** [comment]
Sorry for having this one so long in the cooker, and thanks!

**Comment by pmderodat** [comment]
No problem… thanks for the review!

**Pull request merged**